python/ray/serve/scripts.py

```diff
@@ -286,68 +289,111 @@ def run(
 ):
     sys.path.insert(0, app_dir)
 
-    final_runtime_env = parse_runtime_env_args(
-        runtime_env=runtime_env,
-        runtime_env_json=runtime_env_json,
-        working_dir=working_dir,
-    )
+    # If address is specified through either the command or through the RAY_ADDRESS
```

Comment:
Why don't we use job submission for local clusters as well?

Reply:
We use job submission if the user explicitly specifies an address of a local cluster, otherwise we just connect to it normally through `ray.init()`.

Reply:
It would be better to have a single deployment model for everything if possible. any reason not to always use the same path? (we could `ray.init()` and then use the dashboard SDK for it)

Reply:
Changed so that `serve run` will always use Job Submission.